Frameworks/PyTorch/Getting-Started-Samples/PyTorch_HelloWorld/PyTorch_Hello_World.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+'''
+==============================================================
+ Copyright © 2019 Intel Corporation
+
+ SPDX-License-Identifier: MIT
+==============================================================
+'''
+
+import torch
+import torch.nn as nn
+from torch.utils import mkldnn
+from torch.utils.data import Dataset, DataLoader
+
+'''
+BS_TRAIN: Batch size for training data
+BS_TEST:  Batch size for testing data
+EPOCHNUM: Number of epoch for training
+'''
+BS_TRAIN = 50
+BS_TEST  = 10
+EPOCHNUM = 2
+
+'''
+TestDataset class is inherited from torch.utils.data.Dataset.
+Since data for training involves data and ground truth, a flag "train" is defined in the initialization function. When train is True, instance of TestDataset gets a pair of training data and label data. When it is False, the instance gets data only for inference. Value of the flag "train" is set in __init__ function.
+In __getitem__ function, data at index position is supposed to be returned.
+__len__ function returns the overall length of the dataset.
+'''
+class TestDataset(Dataset):
+    def __init__(self, train = True):
+        super(TestDataset, self).__init__()
+        self.train = train
+
+    def __getitem__(self, index):
+        if self.train:
+            return torch.rand(3, 112, 112), torch.rand(6, 110, 110)
+        else:
+            return torch.rand(3, 112, 112)
+
+    def __len__(self):
+        if self.train:
+            return 100
+        else:
+            return 20
+
+'''
+TestModel class is inherited from torch.nn.Module.
+Operations that will be used in the topology are defined in __init__ function.
+Input data x is supposed to be passed to the forward function. The topology is implemented in the forward function. When perform training/inference, the forward function will be called automatically by passing input data to a model instance.
+'''
+class TestModel(nn.Module):
+    def __init__(self):
+        super(TestModel, self).__init__()
```

Comment:
idem

Reply:
Feedback from sample author:
Hi, since PyTorch codes also use this style, we follow this at this moment.
https://github.com/pytorch/pytorch/search?q=__init__%28%29&unscoped_q=__init__%28%29

Reply:
@keryell 

move the discussion into issue#125.
https://github.com/oneapi-src/oneAPI-samples/issues/125